var/spack/repos/builtin/packages/gptune/package.py

```diff
@@ -43,7 +43,7 @@ class Gptune(CMakePackage):
     depends_on("py-scikit-learn", type=("build", "run"))
     depends_on("py-matplotlib", type=("build", "run"))
     depends_on("py-pyyaml", type=("build", "run"))
-    depends_on("py-scikit-optimize@0.9.0", patches=[patch("space.patch")], type=("build", "run"))
+    depends_on("py-scikit-optimize@0.9.0", type=("build", "run"))
```

Comment:
Did you confirm that this still builds without this patch? I'm guessing it was there for a reason.

Reply:
reverted changes to py-scikit-optimize